components/sandbox/toolchain-host-operator/base/monitoring/sandbox-registration-service-proxy.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
```

Comment:
You added this file but did not included it in any kustomization.yaml. I deployed nothing to stg and prod because of that, not sure this was the intention.

Reply:
Because I'm an idiot 🤦 

Thanks a lot for catching that, I always forget to update/add the `kustomization` file. 

Could you please take a look at https://github.com/redhat-appstudio/infra-deployments/pull/3085 🙏